specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -7,6 +7,48 @@
   },
   "paths": {},
   "definitions": {
+    "StorageBlobAccessTier": {
+      "type": "string",
+      "description": "The access tier of the blob.",
+      "enum": [
+        "Hot",
+        "Cool",
+        "Cold",
+        "Archive",
+        "Default"
+      ],
+      "x-ms-enum": {
+        "name": "StorageBlobAccessTier",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Hot",
+            "value": "Hot",
+            "description": "The blob is in access tier Hot"
+          },
+          {
+            "name": "Cool",
+            "value": "Cool",
+            "description": "The blob is in access tier Cool"
+          },
+          {
+            "name": "Cold",
+            "value": "Cold",
+            "description": "The blob is in access tier Cold"
+          },
+          {
+            "name": "Archive",
+            "value": "Archive",
+            "description": "The blob is in access tier Archive"
+          },
+          {
+            "name": "Default",
+            "value": "Default",
+            "description": "The blob is in access tier Default(Inferred)"
+          }
+        ]
+      }
+    },
     "StorageBlobCreatedEventData": {
```

Comment:
We need the new property here too.

Reply:
Added the new property to StorageBlobCreatedEventData

Reply:
The new properties need to be added to the set of required properties. You can look at the GeneratedSystemEvents file to see what it looks like.

Reply:
Added the properties to required properties.